pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -502,7 +511,11 @@ private static void shutdownPubSub() {
             try {
                 pubSubTaskConnectionService.destroy();
             } catch (PubSubTaskException e) {
-                log.error("Failed to shutdown PubSubTask API", e);
+                if (!SparkBootUtil.isShuttingDown()) {
+                    log.error("Failed to shutdown PubSubTask API", e);
+                } else {
+                    log.info("Exception during shutdown", e);
```

Comment:
I am not very familiar with housekeeping, but shouldn't this also be at an ERROR level?

Reply:
My thinking here is that during shutdown, things are going to be messy, and while a best attempt is made to shut down cleanly, if something blows up during VM termination, it's interesting but not something that we want to consider as a production alert.